.github/workflows/test-single.yml

```
@@ -97,7 +97,7 @@ jobs:
           - name: "3.12"
             file_name: "3.12"
             action: "3.12"
-            apt: "3.12"
+            apt: "3.12.7"
```

and this isn't even used `:]`
```suggestion
            apt: "3.12"
```